packages/tasit-account/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/*
```

Comment:
Would something like `**/node_modules/*` in the root .gitignore work?

Reply:
I think so but I'm not sure, Want you that I test it?

Reply:
It's not a high priority - don't worry about it. Thanks! Just something to keep in mind for the future in case it becomes a pain to keep adding files like this.

Reply:
Done